cmd/snap/cmd_quota.go

```diff
@@ -215,11 +247,32 @@ func (x *cmdSetQuota) parseQuotas() (*client.QuotaValues, error) {
 		quotaValues.Threads = int(value)
 	}
 
+	if x.JournalSizeMax != "" || x.JournalRateLimit != "" {
+		quotaValues.Journal = &client.QuotaJournalValues{}
+		if x.JournalSizeMax != "" {
+			value, err := strutil.ParseByteSize(x.JournalSizeMax)
+			if err != nil {
+				return nil, err
```

Comment:
```suggestion
				return nil, fmt.Errorf("cannot parse journal size %q: %v", x.JournalSizeMax, err)
```

Reply:
Nice suggestions on the error reporting, I've added all of them!